inspirehep/pidstore/__init__.py

```diff
@@ -1,23 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This is not a bash script (in every python file).

Reply:
oups my bad thanks